### DISCUSSION TITLE
How might I figure out how to support ZZZ shader (is this even possible)?

### DISCUSSION THREAD

**Discussion by O-J1** [lvoxx/LSCherry#18] [Q&A] [answered]
I started modding ZZZ and was looking for a shader to emulate how it looks so I have better idea of how it will look ingame.

My question is how might I modify this setup to support/emulate ZZZ?

There is the beginnings of doco around the textures.
https://leotorrez.github.io/modding/guides/zzz-textures#normal-map

**Comment by lvoxx** [comment, marked as answer]
The built-in set for Mihoyo's games focuses on the Lightmap Separator component to apply colors by region appropriately. Currently, in version 1.0.3, I am focusing on optimizing HI3 and vanilla toon. You can create or contribute a built-in set for the ZZZ section of the project.
![image](https://github.com/user-attachments/assets/cb8a54dd-1a8a-4a6e-b21c-216681da6980)
